### PR TITLE
LPS-140227 buildService

### PR DESCRIPTION
**Pull request by marcelabc**
Related issue: https://issues.liferay.com/browse/LPS-140227

### CODE REVIEW

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:test:relevant

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectRelationshipLocalServiceImpl.java [hunk 1]
```diff
@@ -312,6 +312,28 @@ public List<ObjectRelationship> getObjectRelationships(
...
+	@Override
+	public List<ObjectRelationship> getObjectRelationships(
+		long objectDefinitionId1, String deletionType, boolean reverse) {
+
+		return objectRelationshipPersistence.findByODI1_DT_R(
+			objectDefinitionId1, deletionType, reverse);
+	}
+
+	@Override
+	public boolean hasDeletionType(
```
Comment (gabrielwas) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectRelationshipLocalServiceImpl.java:324]: @marcelabc are you using it just for prevent deletion type, right? Maybe we could implement a method `isPreventDeletionType` with a couple of changes. What do you think?

**Review by gabrielwas — CHANGES_REQUESTED** [review]

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectRelationshipLocalServiceImpl.java (hunk 1 above)
Comment (marcelabc) [reply modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectRelationshipLocalServiceImpl.java:324]: @gabrielwas Okay! We just have to keep in mind to refactor to a generic one if it needs another logic involving another deletionType in the future. 

I created the method this way to make the most generic, but I will change it to a specific one :+1:

**Comment by marcelabc** [comment]
ci:test:sf

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectRelationshipLocalServiceImpl.java (hunk 1 above)
Comment (marcelabc) [reply modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectRelationshipLocalServiceImpl.java]: I rename the method, so I'm removing the label "Author Action Required".

**Comment by marcelabc** [comment]
ci:test:relevant

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectRelationshipLocalServiceImpl.java (hunk 1 above)
Comment (gabrielwas) [reply modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectRelationshipLocalServiceImpl.java]: After internal discussion, we've decided to keep the method as it is and just rename the method. Thanks, @marcelabc !

**Comment by marcelabc** [comment]
ci:forward

**Pull request closed without merging**